doc/source/Platforms.rst

```diff
@@ -141,6 +143,14 @@ The following is required for building new spack environments and for using spac
    module use module use /work/noaa/da/jedipara/spack-stack/modulefiles
    module load miniconda/3.9.7
 
+.. note::
+   If the build of `crtm` or another package fails with errors related to ``git lfs not found``, load the ``git`` module and initialize ``git lfs``:
```

Comment:
This shouldn't happen, with the `skip_git_lfs` option. I'm fine with requiring it, but I specifically added that option so you don't have to.

Reply:
Ah, I wonder if that change got lost in the merge.

Reply:
https://github.com/NOAA-EMC/spack/blob/cb1377b6117220e40e7d454e7c5b94e44f20ca53/lib/spack/spack/fetch_strategy.py#L874

Reply:
@kgerheiser 
```
[skylab-1.0.0-gnu-10.2.0] 518 role-da@Orion-login-1 spack-stack-1.0.0-rc1 ❯❯❯ less spack/var/spack/repos/builtin/packages/crtm/package.py
# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
# Spack Project Developers. See the top-level COPYRIGHT file for details.
#
# SPDX-License-Identifier: (Apache-2.0 OR MIT)

from spack import *


class Crtm(CMakePackage):
    """The Community Radiative Transfer Model (CRTM) package.
    The CRTM is composed of four important modules for gaseous transmittance,
    surface emission and reflection, cloud and aerosol absorption and
    scattering, and a solver for a radiative transfer."""

    homepage = "https://www.jcsda.org/jcsda-project-community-radiative-transfer-model"
    git = 'https://github.com/JCSDA/crtm.git'

    maintainers = ['t-brown', 'edwardhartnett', 'kgerheiser', 'Hang-Lei-NOAA']

    depends_on('netcdf-fortran', when='@2.4.0:')

    # ecbuild release v2.4.0 is broken
    # add ecbuild dependency for next release with fix
    # depends_on('ecbuild', when='@2.4.0:', type=('build'))

    # REL-2.4.0_emc (v2.4.0 ecbuild does not work)
    version('2.4.0', commit='a831626', skip_git_lfs=True)
    # Uses the tip of REL-2.3.0_emc branch
    version('2.3.0', commit='99760e6', skip_git_lfs=True)
```
but:
```
==> Installing crtm-2.3.0-gqtqxsqt7g6xeixxjmmfikw7tbjc7c5j
==> No binary for crtm-2.3.0-gqtqxsqt7g6xeixxjmmfikw7tbjc7c5j found: installing from source
git-lfs smudge -- 'src/Build/CMakeFiles/3.17.1/CMakeDetermineCompilerABI_Fortran.bin': git-lfs: command not found
error: external filter git-lfs smudge -- %f failed -1
error: external filter git-lfs smudge -- %f failed
fatal: src/Build/CMakeFiles/3.17.1/CMakeDetermineCompilerABI_Fortran.bin: smudge filter lfs failed
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry the checkout with 'git checkout -f HEAD'

==> Error: FetchError: All fetchers failed for spack-stage-crtm-2.3.0-gqtqxsqt7g6xeixxjmmfikw7tbjc7c5j

/work/noaa/da/role-da/spack-stack/spack-stack-1.0.0-rc1/spack/lib/spack/spack/package.py:1537, in do_fetch:
```